jcl/src/java.base/share/classes/openj9/internal/foreign/abi/LayoutStrPreprocessor.java

```diff
@@ -24,75 +24,67 @@
 
 import java.util.List;
 
-/*[IF JAVA_SPEC_VERSION >= 19]*/
+/*[IF JAVA_SPEC_VERSION >= 20]*/
 import java.lang.foreign.FunctionDescriptor;
 import java.lang.foreign.GroupLayout;
-/*[IF JAVA_SPEC_VERSION == 19]*/
-import java.lang.foreign.MemoryAddress;
-/*[ENDIF] JAVA_SPEC_VERSION == 19 */
 import java.lang.foreign.MemoryLayout;
-/*[IF JAVA_SPEC_VERSION >= 20]*/
 import java.lang.foreign.MemorySegment;
 import java.lang.foreign.PaddingLayout;
-/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
 import java.lang.foreign.SequenceLayout;
 import java.lang.foreign.ValueLayout;
-/*[IF JAVA_SPEC_VERSION >= 20]*/
 import jdk.internal.foreign.abi.LinkerOptions;
-/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
-/*[ELSE] JAVA_SPEC_VERSION >= 19 */
-/*[IF JAVA_SPEC_VERSION <= 17]*/
+/*[ELSE] JAVA_SPEC_VERSION >= 20 */
 import jdk.incubator.foreign.CLinker.TypeKind;
 import static jdk.incubator.foreign.CLinker.TypeKind.*;
-/*[ENDIF] JAVA_SPEC_VERSION <= 17 */
 import jdk.incubator.foreign.FunctionDescriptor;
 import jdk.incubator.foreign.GroupLayout;
 import jdk.incubator.foreign.MemoryAddress;
 import jdk.incubator.foreign.MemoryLayout;
 import jdk.incubator.foreign.SequenceLayout;
 import jdk.incubator.foreign.ValueLayout;
-/*[ENDIF] JAVA_SPEC_VERSION >= 19 */
+/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
 
 /**
  * The methods of the class are used to preprocess the layout specified in the function
  * descriptor of downcall or upcall by removing all unnecessary attributes and converting
  * it to a simplified symbol string.
  */
+@SuppressWarnings("nls")
 final class LayoutStrPreprocessor {
 
-	/*[IF JAVA_SPEC_VERSION <= 17]*/
+	/*[IF JAVA_SPEC_VERSION == 17]*/
 	private static final String VARARGS_ATTR_NAME;
 
 	static {
-		final String osName = System.getProperty("os.name").toLowerCase(); //$NON-NLS-1$
-		final String arch = System.getProperty("os.arch").toLowerCase(); //$NON-NLS-1$
+		final String osName = System.getProperty("os.name").toLowerCase();
+		final String arch = System.getProperty("os.arch").toLowerCase();
 
 		/* Note: the attributes intended for the layout with variadic argument are defined in OpenJDK. */
-		if ((arch.equals("amd64") || arch.equals("x86_64"))) { //$NON-NLS-1$ //$NON-NLS-2$
-			if (osName.startsWith("windows")) { //$NON-NLS-1$
-				VARARGS_ATTR_NAME = "abi/windows/varargs"; //$NON-NLS-1$
+		if ((arch.equals("amd64") || arch.equals("x86_64"))) {
+			if (osName.startsWith("windows")) {
+				VARARGS_ATTR_NAME = "abi/windows/varargs";
 			} else {
-				VARARGS_ATTR_NAME = "abi/sysv/varargs";; //$NON-NLS-1$
+				VARARGS_ATTR_NAME = "abi/sysv/varargs";;
```

Comment:
Please remove the empty statements here and on lines 71 and 73.

Reply:
Fixed.